distribution/test/test-metacard-validator/pom.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
❓ Was all this added code moved from somewhere else?

Reply:
Looks like it came from here: https://github.com/codice/ddf-sdk